<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="allTests" tests="9" assertions="9" errors="5" failures="0" skipped="0" time="0.794088">
    <testsuite name="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" tests="8" assertions="8" errors="4" failures="0" skipped="0" time="0.778518">
      <testcase name="testBuscarDadosNoEsiraEstudante" class="IntegracaoTest" classname="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" line="15" assertions="1" time="0.051555"/>
      <testcase name="testBuscarDadosNoEsiraDisciplina" class="IntegracaoTest" classname="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" line="24" assertions="1" time="0.018499"/>
      <testcase name="testBscarDadosNoEsiraCurso" class="IntegracaoTest" classname="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" line="32" assertions="1" time="0.009463"/>
      <testcase name="testBuscarDadosNoEsiraInscricao" class="IntegracaoTest" classname="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" line="40" assertions="1" time="0.114056"/>
      <testcase name="testListaDeAlunos" class="IntegracaoTest" classname="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" line="49" assertions="1" time="0.043972">
        <error type="PHPUnit\Framework\UnintentionallyCoveredCodeError">Risky Test
</error>
      </testcase>
      <testcase name="testListaDeDisciplinas" class="IntegracaoTest" classname="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" line="59" assertions="1" time="0.038858">
        <error type="PHPUnit\Framework\UnintentionallyCoveredCodeError">Risky Test
</error>
      </testcase>
      <testcase name="testListaDeCursos" class="IntegracaoTest" classname="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" line="68" assertions="1" time="0.024054">
        <error type="PHPUnit\Framework\UnintentionallyCoveredCodeError">Risky Test
</error>
      </testcase>
      <testcase name="testListaDeInscricoes" class="IntegracaoTest" classname="IntegracaoTest" file="C:\xampp\htdocs\sigepauta\tests\IntegracaoTest.php" line="77" assertions="1" time="0.478061">
        <error type="PHPUnit\Framework\UnintentionallyCoveredCodeError">Risky Test
</error>
      </testcase>
    </testsuite>
    <testsuite name="ExameEspecialTest" file="C:\xampp\htdocs\sigepauta\tests\ExameEspecialTest.php" tests="1" assertions="1" errors="1" failures="0" skipped="0" time="0.015570">
      <testcase name="testResult" class="ExameEspecialTest" classname="ExameEspecialTest" file="C:\xampp\htdocs\sigepauta\tests\ExameEspecialTest.php" line="15" assertions="1" time="0.015570">
        <error type="PHPUnit\Framework\UnintentionallyCoveredCodeError">Risky Test
</error>
      </testcase>
    </testsuite>
  </testsuite>
</testsuites>
